cmd/draft/init.go

```diff
@@ -86,10 +76,41 @@ func newInitCmd(out io.Writer, in io.Reader) *cobra.Command {
 	f.BoolVar(&i.autoAccept, "auto-accept", false, "automatically accept configuration defaults (if detected). It will still prompt for information if this is set to true and no cloud provider was found")
 	f.BoolVar(&i.dryRun, "dry-run", false, "go through all the steps without actually installing anything. Mostly used along with --debug for debugging purposes.")
 	f.StringVarP(&i.image, "draftd-image", "i", "", "override Draftd image")
+	// flags for bootstrapping draftd with tls
+	f.BoolVar(&tlsEnable, "draftd-tls", false, "install Draftd with TLS enabled")
+	f.BoolVar(&tlsVerify, "draftd-tls-verify", false, "install Draftd with TLS enabled and to verify remote certificates")
+	f.StringVar(&tlsKeyFile, "draftd-tls-key", "", "path to TLS key file to install with Draftd")
+	f.StringVar(&tlsCertFile, "draftd-tls-cert", "", "path to TLS certificate file to install with Draftd")
+	f.StringVar(&tlsCaCertFile, "tls-ca-cert", "", "path to CA root certificate")
```

Comment:
do we want to make this flag `--draftd-tls-ca-cert` for consistency?

Reply:
It's left general because the CA certificate is common to both draft client and server. Consistency here is with helm tls workflow.

Reply:
:+1: